Web/src/SettingsPage.js

```diff
@@ -228,57 +228,54 @@ export default function SettingsPage(props) {
 
 
     /**
-     * Ensure the log in of the user in the services
-     * @function LogWith
-     * @param {string} serviceName the name of the service where you want to register
+     * Ensure the log in of the user on Spotify
+     * @function spotifyConnexion
```

Comment:
missing async field (it has to be empty)

Reply:
done